src/components/header/examples/census-nisra/index.njk

```diff
@@ -8,9 +8,9 @@ layout: none
     "theme": 'census',
     "header": {
         "logo": "nisra-logo-en",
+        "mobileLogo": "nisra-logo-mobile-en",
```

Comment:
We can't change this file name if we want census to inherit this PR, because it's specified in the services.  Would be a breaking change